packages/app/backend-http-client/src/client/constants.ts

```diff
@@ -29,4 +29,13 @@ export const TEST_OPTIONS: RequestOptions<unknown, any> = {
   responseSchema: z.unknown(),
 }
 
+/**
+ * Technically 204 will send an empty body, but undici-retry defaults to parsing unknown mimetype as text for
+ * compatibility reasons, so we should expect to get an empty string here
+ */
 export const NO_CONTENT_RESPONSE_SCHEMA = z.string().length(0)
+
+/**
+ * This schema is to be used when we don't really care about the response type and are prepared to accept any value
+ */
+export const UNKNOWN_RESPONSE_SCHEMA = z.unknown()
```

Comment:
@kibertoad The on we had was `unknown`, I think it is better than `any` but please let me know if disagree